### ISSUE TITLE
problem of LungInfection-Test datasets

### ISSUE THREAD

**Issue by Yufengevan** [DengPingFan/Inf-Net#10]
I have download the LungInfection datasets from link in the Readme. LungInfection-Test contains  48 pictures and your paper mentioned is 50. The same problem occur in the MultiClassInfection-Test.

**Comment by DengPingFan** [comment]
Hi, actually the original dataset contains 50 images. However, we found there are two images with very small resolution and black ground-truth. Thus, we discard these two images in our training set.

**Comment by DengPingFan** [comment]
If you add these two images in your model, it may introduce a native impact to your model training.

**Comment by Yufengevan** [comment]
Thank your reply. Got it.

**Comment by Yufengevan** [comment]
Sorry to bother you, I have checked data again and you have mentioned two uncomfortable images will have native impact to model training. But, the data set with missing images is the test set rather than the training set. Will this result in better performance?

**Comment by DengPingFan** [comment]
You are right. == Note that ==: In our manuscript, we said that the total testing images are 50. However, we found there are two images with very small resolution and black ground-truth. Thus, we discard these two images in our testing set. The above link only contains 48 testing images.

**Comment by Yufengevan** [comment]
okay. Can you upload the remaining two images of two data sets? I am very interested in your research and want to review your experimental results first. Later, if I submit a paper, I will also cite your article. Thank you so much.

**Comment by DengPingFan** [comment]
Please give me your Email, I will send it to you.

**Comment by DengPingFan** [comment]
Also, note that the evaluation results in our manuscript are based on 48 images.

**Comment by Yufengevan** [comment]
Emm... You means the number of test images in manuscript are 50 and the number in fact experiment are 48?  My Email is : zdhnyf2014@sina.cn

**Comment by DengPingFan** [comment]
Hi, The dataset has been sent by email. Note that we will update the manuscript accordingly. You still need to compare with those models under 48 testing sets.

**Comment by Yufengevan** [comment]
Okay. I have received your email, but i couldn't download the data because of no VPN. Could you directly send the images of two data sets in email when you are in convenience?
Thank you again and have a nice day.

**Comment by DengPingFan** [comment]
Maybe you need a VPN.
Best
